### ISSUE TITLE
IRS supports traversal to upwards direction for SingleLevelUsageAsPlanned Version 2.0.0

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/item-relationship-service#557]
**As** product
**I want** to support upwards direction for asPlanned Lifecycle since SLUAP version 2.0.0
**so that** we are compliant with the 2.0.0 aspect mode standard 

## Link 
- relates to https://github.com/eclipse-tractusx/item-relationship-service/issues/564
- relates to https://github.com/eclipse-tractusx/item-relationship-service/issues/558
- blocked by https://jira.catena-x.net/browse/TDG-47

## Acceptance Criteria
- [ ] Upwards direction for SingleLevelUsageAsPlanned for Version 2.0.0 is supported in IRS
- [ ] https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_usage_as_planned/2.0.0/gen

## Out of Scope

- ...

## LOP 
- [ ] @mkanal Create TDG Ticket requesting for SLUAS Aspect models in version 2.0.0

**Comment by mkanal** [comment]
@mkanal  Create a ticket for new testdata

**Comment by ds-jhartmann** [comment]
Already implemented by #470
